demo/predictions/js/prediction-use-case.js

```diff
@@ -60,64 +12,37 @@ class PredicatedLateUseCase extends UseCase {
     }
 
     _reduceVisibilityOfAlreadyExecutedElements() {
-        this._bpmnVisualization.bpmnElementsRegistry.addCssClasses(this._alreadyExecutedElements, 'state-already-executed');
+        this._bpmnVisualization.bpmnElementsRegistry.addCssClasses(this._dataExecutionManager.executedElements, 'state-already-executed');
     }
 
     _registerInteractions() {
         // on hover, highlight the predicted path
-        const bpmnElements = this._bpmnVisualization.bpmnElementsRegistry.getElementsByIds(this._predictionData.elementWithPrediction.bpmnId);
-        const elementTogglingPath = bpmnElements[0]; // exist and only one
+        const elementTogglingPath = this._bpmnVisualization.bpmnElementsRegistry.getElementsByIds(this._dataExecutionManager.runningElementWithPrediction.bpmnId)[0]; // exist and only one
 
         const highlightPredictedPath = () => {
             this._toggleHighlightRunningElementsWithoutPrediction();
-            this._bpmnVisualization.bpmnElementsRegistry.toggleCssClasses(this._predictionData.predictedPath.ids, this._predictionData.predictedPath.cssClasses);
+            let predictedPath = this._dataExecutionManager.predictedPaths;
```

Comment:
**suggestion:** It looks like `predictedPath` is never changed afterwards.
In addition, there is an inconsisency in names. Here the constant is using singular, whereas in DataExecutionManager, it is using the plural form.
```suggestion
            const predictedPath = this._dataExecutionManager.predictedPaths;
```